java/java.lsp.server/vscode/package.json

```diff
@@ -65,8 +65,8 @@
 		],
 		"debuggers": [
 			{
-				"type": "java-polyglot",
-				"label": "Java (Polyglot)",
+				"type": "java8+",
+				"label": "Java 8+",
```

Comment:
The "Java 8+" name seems to nicely stress the core functionality of Apache NetBeans Language Server VSCode extension - e.g. the ability to run on JDK8 (other extensions require JDK11). I think it is a good name.